modules/mcuboot/boot/zephyr/Kconfig

```diff
@@ -77,12 +77,13 @@ choice BOOT_SIGNATURE_TYPE
 endchoice
 
 choice BOOT_ECDSA_IMPLEMENTATION
-	default BOOT_NRF_EXTERNAL_CRYPTO if SECURE_BOOT
+	default BOOT_NRF_EXTERNAL_CRYPTO
```

Comment:
No?

Reply:
If we can accept that it's a bit harder to enable external crypto we can change it back.

Reply:
```
BOOT_NRF_EXTERNAL_CRYPTO (defined at /home/siho/multi/nrf/modules/mcuboot/boot/zephyr/Kconfig:83), with definition...

config BOOT_NRF_EXTERNAL_CRYPTO
	bool "Use Shared Crypto from bootloader"
	select BOOT_USE_NRF_EXTERNAL_CRYPTO
	select SECURE_BOOT_CRYPTO
	depends on <choice BOOT_ECDSA_IMPLEMENTATION>

...depends on <choice BOOT_ECDSA_IMPLEMENTATION> (defined at /home/siho/multi/nrf/modules/mcuboot/boot/zephyr/Kconfig:79, /home/siho/multi/bootloader/mcuboot/boot/zephyr/Kconfig:96), with definition...

choice BOOT_ECDSA_IMPLEMENTATION
	bool
	default BOOT_NRF_EXTERNAL_CRYPTO if SECURE_BOOT
	default BOOT_ECDSA_CC310 if HAS_HW_NRF_CC310

choice BOOT_ECDSA_IMPLEMENTATION
	bool "Ecdsa implementation"
	default BOOT_ECDSA_TINYCRYPT
	depends on BOOT_SIGNATURE_TYPE_ECDSA_P256 && <choice BOOT_SIGNATURE_TYPE>

...depends on SECURE_BOOT (defined at /home/siho/multi/nrf/subsys/bootloader/Kconfig:25), with definition...
```

Is the circular dependency I got.